### PR TITLE
Make everything an error

### PR DESCRIPTION
**Pull request by Chris-V**
Here is the proposition for #2. I compiled GWTP against this branch successfully: https://github.com/ArcBees/GWTP/tree/cv_arcbees_tools

### CODE REVIEW

File: checkstyle/src/main/resources/checkstyle.xml [hunk 1]
```diff
@@ -22,54 +23,37 @@
         <module name="StringLiteralEquality"/>
         <module name="UnnecessaryParentheses"/>
         <module name="Indentation">
-            <property name="basicOffset" value="4" />
+                <property name="throwsIndent" value="8"/>
```
Comment (spg) [inline checkstyle/src/main/resources/checkstyle.xml]: wrong indent

File: checkstyle/src/main/resources/checkstyle.xml [hunk 2]
```diff
@@ -11,9 +14,7 @@
         <module name="ExplicitInitialization"/>
         <module name="GenericWhitespace"/>
         <module name="MethodParamPad"/>
-        <module name="MissingOverride">
-            <property name="severity" value="error"/>
-        </module>
+        <module name="MissingOverride"/>
```
Comment (spg) [inline checkstyle/src/main/resources/checkstyle.xml]: I think this one should be an error

File: checkstyle/src/main/resources/checkstyle.xml [hunk 3]
```diff
@@ -22,54 +23,37 @@
...
-            <property name="severity" value="info"/>
-        </module>
-        <module name="InterfaceIsType">
-            <property name="severity" value="error"/>
-        </module>
-        <module name="RedundantImport">
-            <property name="severity" value="error"/>
-        </module>
+        <module name="InterfaceIsType"/>
+        <module name="RedundantImport"/>
```
Comment (spg) [inline checkstyle/src/main/resources/checkstyle.xml]: this one also should be an error

File: checkstyle/src/main/resources/checkstyle.xml [hunk 4]
```diff
@@ -22,54 +23,37 @@
...
         <module name="UnusedImports">
```
Comment (spg) [inline checkstyle/src/main/resources/checkstyle.xml]: unused import should be an error

File: checkstyle/src/main/resources/checkstyle.xml (hunk 3 above)
Comment (Chris-V) [inline checkstyle/src/main/resources/checkstyle.xml]: All the checks are errors, inherited from line 6

Comment (spg) [inline checkstyle/src/main/resources/checkstyle.xml]: Gotcha, nice feature

File: checkstyle/src/main/resources/checkstyle.xml (hunk 2 above)
Comment (christiangoudreau) [inline checkstyle/src/main/resources/checkstyle.xml]: /agreed

File: checkstyle/src/main/resources/checkstyle.xml (hunk 4 above)
Comment (christiangoudreau) [inline checkstyle/src/main/resources/checkstyle.xml]: / agreed

**Comment by christiangoudreau** [comment]
LGTM

**Comment by spg** [comment]
LGTM

**Pull request merged**